packages/cli-kit/src/node/cli.ts

```diff
@@ -46,7 +47,18 @@ export async function runCLI(options: RunCLIOptions) {
 
   run(undefined, options.moduleURL)
     .then(flush)
-    .catch((error: Error): Promise<void | Error> => {
+    .catch(async (error: Error): Promise<void | Error> => {
+      if (error instanceof CancelExecution) {
+        info(
+          `✨  ${
+            error.message && error.message !== ''
+              ? error.message
+              : `Executed using CLI ${content`${token.yellow(await constants.versions.cliKit())}`.value}`
+          }`,
+        )
+        process.exit(0)
```

Comment:
Returning here instead of letting the command to finish as we did before will skip the analytics reporting when the command is canceled. But I guess it's not a big deal.

Reply:
Maybe we could let finishing the execution instead of exit with success code.